unittest/factory/integrator.cpp

```diff
@@ -50,17 +50,10 @@ boost::shared_ptr<crocoddyl::IntegratedActionModelAbstract> IntegratorFactory::c
       action = boost::make_shared<crocoddyl::IntegratedActionModelEuler>(model);
       break;
     case IntegratorTypes::IntegratorRK2:
-<<<<<<< HEAD
```

Comment:
It seems you have rebase or something.
Please fix this issue.

Reply:
I have already fixed it, haven't I?

Reply:
Yes, you're right.
Sorry for the noise